js/src/edit.js

```diff
@@ -59,7 +65,10 @@ function registerPlugin() {
 					name="seo-sidebar"
 					title="Yoast SEO"
 				>
-					<p> Contents of the sidebar </p>
+					<YoastSidebarContainer>
+						<p> Contents of the sidebar </p>
+						<CornerstoneToggle onChange={ () => {} } checked={ true } />
```

Comment:
Please make sure to rebase against the latest `trunk` and put this inside the `<Fill>`.